terraform/digitalocean/addons/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "argocd_url" {
+  value = "https://${local.argocd_domain}"
+}
+output "argocd_user" {
+  value = "admin"
```

Comment:
[p3] ArgoCD default user can't be changed in TF configuration?

Reply:
yes can't be changed